tests/test_ann.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import torch.nn as nn
+
+from unittest import TestCase
+from ezyrb import ANN
+
+np.random.seed(17)
+
+def get_xy():
+    npts = 20
+    dinput = 4
+
+    inp = np.random.uniform(-1, 1, size=(npts, dinput))
+    out = np.array([
+        np.sin(inp[:, 0]) + np.sin(inp[:, 1]**2),
+        np.cos(inp[:, 2]) + np.cos(inp[:, 3]**2)
+    ]).T
+
+    return inp, out
+
+class TestANN(TestCase):
+    def test_constructor_empty(self):
+        ann = ANN()
+
+    def test_fit_mono(self):
+        x, y = get_xy()
+        ann = ANN()
+        ann.fit(x[:, 0].reshape(len(x),1), y[:, 0].reshape(len(y),1))
```

Comment:
just a trick: instead of `reshape(len(x),1)` use `reshape(-1, 1)`. With -1 it will compute the length by itself.

Reply:
Thanks for the useful hint!